android/runtime/common/src/js/titanium.js

```diff
@@ -1,25 +1,21 @@
 /**
  * Appcelerator Titanium Mobile
-
- * Copyright (c) 2011-2013 by Appcelerator, Inc. All Rights Reserved.
+ *
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
+/* global kroll */
+'use strict';
```

Comment:
@garymathews So I think adding this line actually is causing the build failure where it's complaining about line 77 here trying to assign a value to `this.currentActivity`. Toggling this off locally does "fix" it.

Which means this has been silently failing forever. Do we still need these wrapper types?

Do we need to make the property readwrite? It's defined as read-only in our proxy here: https://github.com/appcelerator/titanium_mobile/blob/master/android/modules/android/src/java/ti/modules/titanium/android/AndroidModule.java#L599